asset_dashboard/endpoints.py

```diff
@@ -64,7 +64,7 @@ def model_cls(self):
             'trails': {'model': TrailsInfo},
             'points_of_interest': {'model': PoiInfo},
             'picnic_groves': {'model': PicnicGroves},
-            'parking_lots': {'model': PoiInfo, 'check_for_not_null': True}
```

Comment:
removed this because it's an `AND` query and we needed to add some more `AND`s to other models. so i made some additions to the model.Search subclass. see related changes in this endpoints.py file and the models.py file.